packages/transformer/src/IModelTransformer.ts

```diff
@@ -638,6 +635,10 @@ export class IModelTransformer extends IModelExportHandler {
       skipPropagateChangesToRootElements:
         options?.skipPropagateChangesToRootElements ?? true,
     };
+    // check if authorization client is defined
+    if (IModelHost.authorizationClient === undefined) {
+      Logger.logWarning(loggerCategory, "No authorization client provided");
```

Comment:
[nit pick] Warning could be more descriptive.  :
- Where auth client is not provided? Now it sounds like I need to provide it as ctor parameter. But in reality it is not set in iModelHost.